homeassistant/components/gogogate2/cover.py

```diff
@@ -55,64 +55,42 @@ def __init__(
         """Initialize the object."""
         unique_id = cover_unique_id(config_entry, door)
         super().__init__(config_entry, data_update_coordinator, door, unique_id)
-        self._api = data_update_coordinator.api
-        self._is_available = True
+        self._attr_supported_features = SUPPORT_OPEN | SUPPORT_CLOSE
+        self._attr_device_class = (
+            DEVICE_CLASS_GATE if self.door.gate else DEVICE_CLASS_GARAGE
+        )
```

Comment:
Is there a special reason for that we decided to use `_attr_supported_features` and `_attr_device_class` instead of keep using the properties? In my opinion using the properties makes the code more clear and readable, but maybe that's more a personal taste

Reply:
That's where the code base is going https://developers.home-assistant.io/docs/core/entity/#entity-class-or-instance-attributes